packages/webapp/src/Routes.jsx

```diff
@@ -316,6 +316,114 @@ const UnknownRecord = React.lazy(() =>
   import('./containers/ErrorHandler/UnknownRecord/UnknownRecord'),
 );
 
+const FinancesRoutes = () => (
+  <Switch>
+    <Route path="/finances/transactions" exact component={Finances} />
+    <Route path="/finances/actual_revenue" exact component={ActualRevenue} />
+    <Route
+      path="/finances/estimated_revenue/plan/:management_plan_id"
+      exact
+      component={UpdateEstimatedCropRevenue}
+    />
+    <Route path="/finances/revenue_types" exact component={RevenueTypes} />
+    <Route path="/finances/add_sale" exact component={AddSale} />
+    <Route path="/finances/manage_custom_revenues" exact component={ManageRevenueTypes} />
+    <Route path="/finances/revenue/:sale_id/" exact component={RevenueDetail} />
+    <Route path="/finances/revenue/:sale_id/edit" exact component={RevenueDetail} />
+    <Route path="/finances/estimated_revenue" exact component={EstimatedRevenue} />
+    <Route path="/finances/labour" exact component={Labour} />
+    <Route path="/finances/other_expense" exact component={OtherExpense} />
+    <Route path="/finances/expense/:expense_id/" exact component={ExpenseDetail} />
+    <Route path="/finances/expense/:expense_id/edit" exact component={ExpenseDetail} />
+    <Route path="/finances/expense_categories" exact component={ExpenseCategories} />
+    <Route path="/finances/add_expense" exact component={AddExpense} />
+    <Route path="/finances/manage_custom_expenses" exact component={ManageExpenseTypes} />
+    <Route path="/finances/add_custom_expense" exact component={AddCustomExpense} />
+    <Route
+      path="/finances/readonly_custom_expense/:expense_type_id"
+      exact
+      component={ReadOnlyCustomExpense}
+    />
+    <Route
+      path="/finances/edit_custom_expense/:expense_type_id"
+      exact
+      component={EditCustomExpense}
+    />
+    <Route path="/finances/add_custom_revenue" exact component={AddCustomRevenue} />
+    <Route
+      path="/finances/readonly_custom_revenue/:revenue_type_id"
+      exact
+      component={ReadOnlyCustomRevenue}
+    />
+    <Route
+      path="/finances/edit_custom_revenue/:revenue_type_id"
+      exact
+      component={EditCustomRevenue}
+    />
+    {/* Redirect old routes to nested finances routes for backwards compatibility */}
```

Comment:
This makes sense but are there specific links we are trying to preserve? Our previous strategy has been to just delete them. Or would it make sense to make a separate file like "reitiredRoutes" if we keep?

Reply:
My thinking was just ensuring that if someone has a page open it would still work for them, but I liked splitting these into a different file so we can eventually move away from them. If we do remove them I'd also need to check the rest of the files to ensure all the links work properly, which you're right we'll have to do eventually to clean this up

Reply:
True, I am not too worried about a person on that page. But I understand it is a lot of work to fix all the broken links...